core/src/components/item/item.ionic.scss

```diff
@@ -1,121 +1,66 @@
-    * @prop --padding-end: Right padding if direction is left-to-right, and left padding if direction is right-to-left of the item
-    * @prop --padding-start: Left padding if direction is left-to-right, and right padding if direction is right-to-left of the item
-    * @prop --padding-top: Top padding of the item
-    *
-    */
   --background: #{globals.$ionic-color-base-white};
   --background-activated: #{globals.$ionic-color-neutral-200};
   --border-color: #{globals.$ionic-color-neutral-300};
   --border-style: #{globals.$ionic-border-style-solid};
-  --border-width: #{globals.$ionic-border-size-025};
+  --border-width: #{0 0 globals.$ionic-border-size-025 0};
   --min-height: #{globals.$ionic-scale-1800};
   --padding-top: #{globals.$ionic-space-200};
   --padding-end: #{globals.$ionic-space-400};
   --padding-bottom: #{globals.$ionic-space-200};
   --padding-start: #{globals.$ionic-space-400};
 
-  @include globals.font-smoothing;
-
-  display: flex;
-
-  position: relative;
-
-  align-items: center;
-  justify-content: space-between;
-
-  outline: none;
-
   font-family: globals.$ionic-font-family;
-
-  text-align: initial;
-  text-decoration: none;
-
-  overflow: hidden;
-  box-sizing: border-box;
 }
 
-// Item: Structure
+// Inner Item
 // --------------------------------------------------
-.item-native {
-  @include globals.padding(var(--padding-top), var(--padding-end), var(--padding-bottom), var(--padding-start));
-  display: flex;
-
-  align-items: center;
-
-  width: 100%;
-
-  min-height: var(--min-height);
-
-  border-bottom-width: var(--border-width);
-  border-bottom-style: var(--border-style);
-  border-bottom-color: var(--border-color);
-
-  background: var(--background);
-
-  overflow: inherit;
-  box-sizing: border-box;
 
-  slot[name="start"]::slotted(*) {
-    @include globals.margin(null, #{globals.$ionic-space-400}, null, null);
-  }
-
-  slot[name="end"]::slotted(*) {
-    @include globals.margin(null, null, null, #{globals.$ionic-space-400});
-  }
+.item-inner {
+  // This prevents the padding/margin from
+  // being excluded from the height calculation.
+  min-height: initial;
+}
 
-  .item-inner,
-  .item-inner .input-wrapper {
-    display: flex;
+// Ionic Item Slots
+// --------------------------------------------------
 
-    flex: 1;
+slot[name="start"]::slotted(*) {
+  @include globals.margin(null, #{globals.$ionic-space-400}, null, null);
+}
 
-    overflow: inherit;
-    box-sizing: border-box;
-  }
+slot[name="end"]::slotted(*) {
+  @include globals.margin(null, null, null, #{globals.$ionic-space-400});
 }
 
 // Item: Disabled
 // --------------------------------------------------
-:host(.item-disabled) {
-  cursor: default;
-  pointer-events: none;
-
-  &::after {
-    @include globals.position(0, 0, 0, 0);
-
-    position: absolute;
 
-    background: #{globals.$ionic-state-disabled};
-
-    content: "";
-  }
+:host(.item-disabled) {
+  opacity: 0.6;
```

Comment:
Can't we use `:host(.item-disabled) {@include globals.disabled-state();}` here?

Reply:
@BenOsodrac the disabled state on Figma only has a opacity change. `disabled-state()` changes the background color, which doesn't match Figma.

Reply:
But it is a rgba one from the tokens: `rgba(255, 255, 255, 0.6))` which matches the #ffffff hex color on figma that is also using the disabled token:

<img width="1412" alt="Screenshot 2024-07-09 at 09 06 47" src="https://github.com/ionic-team/ionic-framework/assets/32780808/1d5b6462-1a24-481a-be40-40843e18bf5a">

Reply:
I looked into this more. The `disabled-state()` mixin won’t work because it [targets](https://github.com/ionic-team/ionic-framework/blob/c48b33b360c57f32d756a4cd496382a441170653/core/src/themes/ionic/ionic.mixins.scss#L20) `background-color` but `item` uses a variable, `--background`, to [handle](https://github.com/ionic-team/ionic-framework/blob/70bd41f65d2fd2bcc6e0671c2b71106edd98756a/core/src/components/item/item.common.scss#L192) it. By setting, `background-color`, it never applies the background color correctly.

We could set `--background: rgba(255, 255, 255, 0.6)` but this also doesn’t work because it changes the background color for the `.item-native` [child element](https://github.com/ionic-team/ionic-framework/blob/70bd41f65d2fd2bcc6e0671c2b71106edd98756a/core/src/components/item/item.common.scss#L157). The opacity needs to be set on the `ion-item`. The only reason that it looks like it’s working is because opacity is already being [set](https://github.com/ionic-team/ionic-framework/blob/70bd41f65d2fd2bcc6e0671c2b71106edd98756a/core/src/components/item/item.common.scss#L150) through `item.common`.

![Screenshot 2024-07-09 at 8 39 21 AM](https://github.com/ionic-team/ionic-framework/assets/13530427/dd831dee-54bb-4ae2-888b-020981595f5d)

Reply:
Alright, I'm convinced! :)

Thanks for looking into, maybe once we gather more different implementations of disabled states we can see it better how to improve the mixin or create a new one 👍